docs/content/direct/common-setup-core-chart.md

```diff
@@ -41,22 +41,18 @@ Remember to include the option `--set "kubeflex-operator.isOpenShift=true"`, whe
 
 ## Obtain the kubeconfigs to access the KubeFlex Control Planes
 
-After the KubeStellar Core has fully deployed, `kflex` CLI can be used to retrieve the kubeconfig of all the KubeFlex Control Planes and store them as contexts of the current kubeconfig:
+After the KubeStellar Core has fully deployed, before changing the kubeconfig context, `kflex` CLI can be used to retrieve the kubeconfig of all the KubeFlex Control Planes and store them as contexts of the current kubeconfig:
 
 ```shell
-echo "Getting the kubeconfig of all Control Planes..."
-for cpname in `kubectl get controlplane -o name`; do
-  cpname=${cpname##*/}
-  echo "Getting the kubeconfig of Control Planes \"$cpname\"..."
-  kflex ctx $cpname
-done
+kflex ctx its1
+kflex ctx wds1
 kflex ctx # switch back to the initial context
 ```
 
-Afterwards the content of a Control Plane `$cpname` can be accessed by specifing its context:
+Afterwards the content of a Control Plane `<cpname>` can be accessed by specifing its context:
 
 ```shell
-kubectl --context "$cpname" ...
+kubectl --context <cpname> ...

```

Comment:
FYI, I am not a fan of using something that is not shell syntax in an exhibited shell command.